src/Console/Output/ProcessOutput.php

```diff
@@ -59,14 +59,14 @@
     private $processedFiles = 0;
 
     /**
-     * @var int|null
+     * @var null|int
      */
     private $symbolsPerLine;
 
     /**
      * @param OutputInterface $output
      * @param EventDispatcher $dispatcher
-     * @param int|null        $nbFiles
+     * @param null|int        $nbFiles
```

Comment:
Done: #2842.

I tried applying the fixer to 2.3 branch as well: only one fix. Is it worth a PR for it?